config/src/main/java/org/springframework/security/config/annotation/web/configuration/WebSecurityConfiguration.java

```diff
@@ -64,15 +62,13 @@
  * @since 3.2
  */
 @Configuration
-public class WebSecurityConfiguration implements ImportAware, BeanClassLoaderAware {
```

Comment:
Same reason as above re: Semantic Versioning

Reply:
This change isn't neccessary, I did not consider the configurations that are extending this one. Will revert it back to implementing `BeanClassLoaderAware`.